include/h2o/mruby_.h

```diff
@@ -177,10 +182,18 @@ mrb_value h2o_mruby_http_join_response_callback(h2o_mruby_context_t *ctx, mrb_va
                                                 int *next_action);
 mrb_value h2o_mruby_http_fetch_chunk_callback(h2o_mruby_context_t *ctx, mrb_value receiver, mrb_value input,
                                               int *next_action);
+mrb_value h2o_mruby_channel_shift_callback(h2o_mruby_context_t *ctx, mrb_value receiver, mrb_value input,
+                                              int *next_action);
```

Comment:
I agree this idea, because it's a little complex to deal with queue in C.
Then I fixed it like below. (I'll add rb file and use script to create embedded code soon.)
https://github.com/h2o/h2o/pull/1336/commits/064a44b6bff761154b224e4b3c253553d4d69fd8

・For ```_h2o__channel_wait```, it seems we can't call fiber yield ruby code from C, then I wrote to call ```_h2o__channel_wait``` directly.
・For initializing Channel object, maybe there is better or more simple way, then I'm willing to fix it.